src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
```

Comment:
Ensure that the `user` prop is correctly passed to the `UserInfo` component. This prop should contain `name` and `email` fields.